test/utils/userOp.ts

```diff
@@ -402,3 +402,44 @@ export async function makeEcdsaModuleUserOpWithPaymaster(
   return userOpWithPaymasterData;
 }
 
+
+export async function makeSARegistryModuleUserOp(
```

Comment:
I like that you decided to introduce a helper function for such kind of userOps

Reply:
:)